core/frontend/src/tile/map/ImageryTileTree.ts

```diff
@@ -228,12 +233,18 @@ class ImageryMapLayerTreeSupplier implements TileTreeSupplier {
   public compareTileTreeIds(lhs: ImageryMapLayerTreeId, rhs: ImageryMapLayerTreeId): number {
     let cmp = compareStrings(lhs.settings.url, rhs.settings.url);
     if (0 === cmp) {
-      cmp = compareBooleans(lhs.settings.transparentBackground, rhs.settings.transparentBackground);
+      cmp = compareStringsOrUndefined(lhs.settings.userName, rhs.settings.userName);

```

Comment:
Do the contents of two otherwise-identical tile trees differ if the user names and/or passwords don't match?

Reply:
Not necessarily, but if we dont include username/password, the tile tree will never be refreshed if credentials have been provided (a layer missing credentials can be attached), or updated.